docs/docs-content/registries-and-packs/registries-and-packs.md

```diff
@@ -13,19 +13,19 @@ sidebar_custom_props:
 # Packs
 
 
-A **Cluster Profile** is made up of preconfigured layers, each of which is called a pack.  In other words, **Packs** are the building block of a cluster profile to create layers such as operating systems, Kubernetes, network, storage, and add-ons. Packs can be broadly categorized into two types:
+A **Cluster Profile** is made up of layers, each of which is created using a pack.  In other words, **Packs** are the building block of a cluster profile to create layers such as operating systems, Kubernetes, network, storage, and add-ons. Packs can be broadly categorized into two types:
```

Comment:
I suggest omitting the second sentence that starts "In other words...". The first sentence says it clearly, and the definition of Infrastructure packs is below.